### PR TITLE
style(tracking): expand page event tracking type

### PR DESCRIPTION
**Pull request by borisonr**
### Overview
not sure if a typescript edit is called style but couldn't figure out which one it should be

we need to add a new page event for marketing, so gotta update the types...
see https://github.com/codecademy-engineering/Codecademy/pull/22633

### PR Checklist

- [ ] Related to designs:
- [x] Related to JIRA ticket: [GROW-2141](https://codecademy.atlassian.net/browse/GROW-2141)
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

File: packages/tracking/src/types.ts [hunk 1]
```diff
@@ -229,3 +232,8 @@ export declare type PagePathVisitedData = BaseEventData & {
   path_id: string;
   path_full_title: string;
 };
+
+export declare type CoursePageVisitedData = BaseEventData & {
```
Comment (JoshuaKGoldberg) [inline packages/tracking/src/types.ts:236]: Oh I just noticed the `declare` above. It's redundant; these types are already being "declared".

```suggestion
export type CoursePageVisitedData = BaseEventData & {
```

**Review by JoshuaKGoldberg — APPROVED** [review]

File: packages/tracking/src/types.ts (hunk 1 above)
Comment (borisonr) [reply packages/tracking/src/types.ts:236]: oh lol copy pasta ftw

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/tracking@0.5.1-alpha.ef8e9f.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60401a5d5b06ef0aac38f026--gamut-preview.netlify.app>https://60401a5d5b06ef0aac38f026--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60401a5d5b06ef0aac38f026>Deploy Logs</a></p>

File: packages/tracking/src/types.ts [hunk 2]
```diff
@@ -51,9 +51,12 @@ export type EventDataTypes = {
     query: BaseEventAnyData;
     result: BaseEventAnyData;
   };
+  // page events for email marketing
+  // potentially to be removed once we clean up user_visit event
```
Comment (neilzo) [inline packages/tracking/src/types.ts:55]: Here's the tech debt ticket for ref: https://codecademy.atlassian.net/browse/MKT-81

**Review by neilzo — APPROVED** [review]

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60410e7163a15e47fcb052df--gamut-preview.netlify.app>https://60410e7163a15e47fcb052df--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60410e7163a15e47fcb052df>Deploy Logs</a></p>

**Pull request merged**